vue/src/TagmanagerTrackingCode/TagManagerTrackingCode.less

```diff
@@ -0,0 +1,11 @@
+div[anchor='tagmanager'] div[matomo-tagmanager-tracking-code] {
+  .list-style-decimal {
+    list-style-type: decimal;
+    list-style-position: inside;
+  }
+
+  .tagManagerTrackingCode .row {
```

Comment:
If this is making the code look nicer, should it be an enhancement in core?

Reply:
This is pretty specific to this view, that's why I scoped it narrowly.